src/test/java/com/hillel/items_exchange/controller/UserControllerTest.java

```diff
@@ -133,4 +134,86 @@ private ResultActions getResultActions(HttpMethod httpMethod, String path,
                 .content(asJsonString(dto));
         return mockMvc.perform(builder).andExpect(matcher);
     }
+
+    @Test
+    @WithMockUser(username = "admin")
+    @DataSet("database_init.yml")
+    void getChildren_Success() throws Exception {
+        mockMvc.perform(get("/user/child/")
+                .contentType(MediaType.APPLICATION_JSON)
+                .accept(MediaType.APPLICATION_JSON))
+                .andDo(print())
+                .andExpect(status().isOk())
+                .andExpect(jsonPath("$", hasSize(2)))
+                .andExpect(jsonPath("$[0].id").value(1L))
+                .andExpect(jsonPath("$[0].sex").value("male"))
+                .andExpect(jsonPath("$[1].id").value(2L))
+                .andExpect(jsonPath("$[1].sex").value("female"));
+
+    }
+
+    @Test
+    @WithMockUser(username = "admin")
+    @Transactional
+    @DataSet("database_init.yml")
+    void addChildren_Success() throws Exception {
+        mockMvc.perform(post("/user/child/")
+                .contentType(MediaType.APPLICATION_JSON)
+                .content(ChildDtoCreatingUtil.getValidChildDtoForCreate())
+                .accept(MediaType.APPLICATION_JSON))
+                .andDo(print())
+                .andExpect(status().isOk());
+    }
+
+    @Test
+    @WithMockUser(username = "admin")
+    @Transactional
+    @DataSet("database_init.yml")
+    @ExpectedDataSet(value = "children/delete.yml")
+    void removeChild_Success() throws Exception {
+        mockMvc.perform(delete("/user/child/")
+                .contentType(MediaType.APPLICATION_JSON)
+                .param("childrenIdToRemove", "1")
+                .accept(MediaType.APPLICATION_JSON))
+                .andDo(print())
+                .andExpect(status().isOk());
+    }
+
+    @Test
+    @WithMockUser(username = "admin")
+    @Transactional
+    @DataSet("database_init.yml")
+    @ExpectedDataSet(value = "children/update.yml")
+    void updateChild_Success() throws Exception {
+        mockMvc.perform(put("/user/child/")
+                .contentType(MediaType.APPLICATION_JSON)
+                .content(ChildDtoCreatingUtil.getValidChildDtoForUpdate())
+                .accept(MediaType.APPLICATION_JSON))
+                .andDo(print())
+                .andExpect(status().isOk());
+    }
+
+    @Test
+    @WithMockUser(username = "admin")
+    @DataSet("database_init.yml")
+    void addChild_Fail_NotValidDto_IdNotZero() throws Exception {
```

Comment:
Don't afraid to create long name of the test or to add DisplayName annotation with more readable description.